w08h03/test/pgdp/datastructures/UnitTest.java

```diff
@@ -78,14 +78,6 @@ public void numericTest() {
 		});
 	}
 
-	// private static int seedDifference = 0;
-	// private static Stream<Arguments> testArray() {
-	// 	// TODO improve this
-	// 	return Stream.generate(() -> {
-	// 		return arguments((new Random(seed + seedDifference++)).ints(seedDifference, 0, Integer.MAX_VALUE).toArray());
-	// 	}).limit(10);
-	// }
```

Comment:
Just a general note for the next time: It's better to move changes like this in a separate PR to keep the PR clean and simple 👍 For example, when we want to revert this PR after merging change with `git revert`, we would also revert small changes like this.

(same applies to the other changes in `UnitTest.java`)

Reply:
Another reason is that it avoids merge conflict like the one that happened right now (#174 removed the lines and Git doesn't know what to choose) 😅

Reply:
Another reason is that it avoids merge conflict like the one that happened right now (#174 removed the lines and Git doesn't know what to choose) 😅

Reply:
Alright :D thanks I'll do that next time